Sources/TripKitUI/managers/TKUITripMonitorManager.swift

```diff
@@ -105,7 +105,7 @@ public class TKUITripMonitorManager: NSObject {
   }
   
   private func identifier(from trip: Trip) -> String? {
-    guard let identifier = trip.saveURL?.lastPathComponent
+    guard let identifier = trip.saveURL?.lastPathComponent ?? trip.tripId
```

Comment:
was this placed as secondary identifier due to it being new?

Reply:
The question. Generally tripId might be preferred, but backend docs currently mark it as optional and indicate it might change, so I thought saveURL is more stable, if it exists. Will clarify with the backend team though.

Reply:
For reference: https://redmine.buzzhives.com/issues/18733